staging/src/k8s.io/client-go/util/retry/util.go

```diff
@@ -42,21 +42,8 @@ var DefaultBackoff = wait.Backoff{
 	Jitter:   0.1,
 }
 
-// OnError executes the provided function repeatedly, retrying if the server returns a specified
-// error. Callers should preserve previous executions if they wish to retry changes. It performs an
-// exponential backoff.
-//
-//     var pod *api.Pod
-//     err := retry.OnError(DefaultBackoff, errors.IsConflict, func() (err error) {
-//       pod, err = c.Pods("mynamespace").UpdateStatus(podStatus)
-//       return
-//     })
-//     if err != nil {
-//       // may be conflict if max retries were hit
-//       return err
-//     }
-//     ...
-//
+// OnError is an abstraction of RetryOnConflict, which allows the caller to control which
```

Comment:
I think we don't need to mention "OnError is an abstraction of RetryOnConflict".

How about:

I will just say OnError allows caller to retry fn in case the error returned by fn is retriable according to errorFunc. backoff defines the maximum retries and the wait interval between two retries.